### PR TITLE
Optimize BEV map rendering to cut iteration time

### PR DESCRIPTION
**Pull request by LukaHenig**
## Summary
- avoid repeatedly converting BEV map logits to NumPy during training and evaluation
- render BEV map overlays directly on the tensor device to reduce per-iteration overhead

## Testing
- python -m compileall nuscenes_data.py train.py train_DDP.py eval.py vis_eval.py

------
https://chatgpt.com/codex/tasks/task_e_68fa5649155c83229c5e802698a70a65

### CODE REVIEW

**Comment by LukaHenig** [comment]
not functionall
